mailer.js

```diff
@@ -165,5 +166,81 @@ module.exports = function (log) {
     return this.send(email)
   }
 
+  Mailer.prototype.passwordChangedEmail = function (message) {
+    log.trace({ op: 'mailer.passwordChangedEmail', email: message.email, uid: message.uid })
+    var query = {
+      email: message.email
+    }
+    var link = this.initiatePasswordResetUrl + '?' + qs.stringify(query)
+
+    var translator = this.translator(message.acceptLanguage)
+    var values = {
+      translator: translator,
+      resetLink: link
+    }
+    var localized = this.templates.passwordChangedEmail(values)
+    var email = {
+      sender: this.sender,
+      to: message.email,
+      subject: translator.gettext('Your Firefox Account password has been changed'),
+      text: localized.text,
+      html: localized.html,
+      headers: {
+        'Content-Language': translator.language,
+        'X-Link': link
+      }
+    }
+    return this.send(email)
+  }
+
+  Mailer.prototype.passwordResetEmail = function (message) {
+    log.trace({ op: 'mailer.passwordiResetEmail', email: message.email, uid: message.uid })
+    var query = {
+      email: message.email
+    }
+    var link = this.initiatePasswordResetUrl + '?' + qs.stringify(query)
+
+    var translator = this.translator(message.acceptLanguage)
+    var values = {
+      translator: translator,
+      resetLink: link
+    }
+    var localized = this.templates.passwordResetEmail(values)
+    var email = {
+      sender: this.sender,
+      to: message.email,
+      subject: translator.gettext('Your Firefox Account password has been reset'),
+      text: localized.text,
+      html: localized.html,
+      headers: {
+        'Content-Language': translator.language,
+        'X-Link': link
+      }
+    }
+    return this.send(email)
+  }
```

Comment:
There's a lot of structural dupliction in these functions.  @shane-tomlinson do you think there's scope for factoring out a common helper here?  (Maybe in a follow-up PR, but worth thinking about)